base/metrics/persistent_memory_allocator.cc

```diff
@@ -981,6 +987,7 @@ LocalPersistentMemoryAllocator::AllocateLocalMemory(size_t size,
                                                     base::StringPiece name) {
   void* address;
 
+#if !defined(STARBOARD)
```

Comment:
Why do we disable this section?

Reply:
This is disabled today in trunk as well as the other part in DeallocateLocalMemory. I'm guessing that's for a reason... So far if I don't disable this part, everything works fine. I can revert this part until we identify what's this for?